src/Aurora.Infrastructure/Scrapers/ScraperBase.cs

```diff
@@ -43,11 +43,29 @@ private void LogRun(ExtendedSearchResult searchResult, System.TimeSpan ranFor)
 
         public async Task<ExtendedSearchResult> PerformSearch(SearchRequest request, CancellationToken token)
         {
-            ValueOrNull<SearchResult> result;
+            ValueOrNull<SearchResult> result = new();
+
             ScraperStatusCode code;
             try
             {
-                result = await SearchInner(request, token);
+                if (request.SearchOptions.Contains(SearchOption.Video))
+                {
+                    var response = await SearchVideosInner(request, token);
+                    result.Value.Items.AddRange(response.Value?.Items);
```

Comment:
Please, don't user this in this way. Create a list of items and than convert it to ValueOrNull